src/badge.py

```diff
@@ -434,7 +446,7 @@ def sync_timestamp(self):
                     self.sendIdentifyReq(10)
 
                 if self.dlg.timestamp_sec != 0:
-                    self.logger.info("Badge datetime was: {},{}".format(self.dlg.timestamp_sec, self.dlg.timestamp_ms))
+                    self.logger.info("Badge datetime was: {},{} and Badge id : {} , and project id : {}".format(self.dlg.timestamp_sec, self.dlg.timestamp_ms,self.badge_id,self.project_id))
```

Comment:
the new additions (printing the project_id and badge_id) to the log are confusing. This specific log line shows the status we got back from the badge. If you want to print the IDs that you sent to the badge, add another log line